source/index.txt

```diff
@@ -1,10 +1,102 @@
-==================
-Kotlin Sync Driver
-==================
+==========================
+{+driver-long+}
+==========================
 
-.. default-domain:: mongodb
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
 
-Your words here. Don't forget to add a toctree!
+   /quick-start
+   /quick-reference
+   /whats-new
+   /usage-examples
+   /faq
+   /connection-troubleshooting
+   /issues-and-help
+   /compatibility
+   View the Source <https://github.com/mongodb/mongo-java-driver>
```

Comment:
C: Source link should be changed to kotlin sync driver

Reply:
Will refer this to Rea – the Kotlin docs also link to this driver, as this is where both Kotlin coroutine and sync driver source lives.

Reply:
Maybe link to the specific folder of the [kotlin sync driver](https://github.com/mongodb/mongo-java-driver/tree/master/driver-kotlin-sync) within the java repo?